plugins/ccxtExchange_test.go

```diff
@@ -556,24 +557,17 @@ func TestCancelOrder_Ccxt(t *testing.T) {
 }
 
 func TestGetOrderConstraints_Ccxt_Precision(t *testing.T) {
-	// coinbasepro gives incorrect precision values so we do not test it here
 	testCases := []struct {
 		exchangeName       string
 		pair               *model.TradingPair
 		wantPricePrecision int8
 		wantVolPrecision   int8
 	}{
 		{
-			// disable ccxt-kraken based tests for now because of the 403 Forbidden Security check API error
```

Comment:
Let’s leave this comment in here too. It’s a TODO for me but I forgot to mark as such.
Can prefix with // TODO NS